### PR TITLE
New/radio button tile

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: src/fhi/blocks/_fhi-form-check-tile.block.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.fhi-form-check-tile {
+    margin-bottom: $fhi-core-space-3;
+    max-width: 25rem;
```
Comment (yleaxeman) [inline src/fhi/blocks/_fhi-form-check-tile.block.scss:3]: @proand - Not sure about this. Should we allow full width?

Comment (proand) [reply src/fhi/blocks/_fhi-form-check-tile.block.scss:3]: Hmm... full width feels like the most systematic aproach to me, even though we are pushing som extra work onto the consumer in makin grids etc... I'm 70/30 in favor of full widht ;)

Comment (yleaxeman) [reply src/fhi/blocks/_fhi-form-check-tile.block.scss:3]: Full width it is

**Review by proand — APPROVED** [review]
LGTM 👍

**Pull request merged**